Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/FitGaussianPeaksTest.py

```diff
@@ -514,6 +532,54 @@ def test_algorithm_does_not_throw_an_error_when_no_valid_peaks_fitted(self):
         self.assertEqual(mtd["peak_table"].rowCount(), 0)
         self.assertEqual(mtd["refit_peak_table"].rowCount(), 0)
 
+    def test_algorithm_uses_right_fit_window(self):
+        ws = CreateWorkspace([0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22] * 2,
+                             [0, 0, 0, 0, 1, 7, 0, 0, 0, 0, 10, 7] + [0] * 12, NSpec=2)
+
+        table = CreateEmptyTableWorkspace()
+        table.addColumn("float", "Centre")
+        table.addRow([20])
+        with mock.patch('plugins.algorithms.WorkflowAlgorithms.FitGaussianPeaks.FitGaussianPeaks.'
+                        'estimate_single_parameters') as mock_estimate_params:
+            mock_estimate_params.return_value = None
+            FitGaussianPeaks(InputWorkspace=ws,
+                             PeakGuessTable=table,
+                             EstimateFitWindow=False,
+                             FitWindowSize=11)
+            correct_x_val = np.array([0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22])
+            correct_y_val = np.array([0, 0, 0, 0, 1, 7, 0, 0, 0, 0, 10, 7])
+
+            arguements = mock_estimate_params.call_args_list[0][0]
+            np.testing.assert_equal(arguements[0], correct_x_val)
```

Comment:
why are we using`np.testing` and not the normal testing?

Reply:
I am using np.testing as some of the arguements are numpy arrays

Reply:
https://stackoverflow.com/questions/7473071/unittests-assertequal-and-iterables-only-check-the-contents looks like you can use  `assertCountEqual(a,b)`